src/applications/coronavirus-vaccination/components/Introduction.jsx

```diff
@@ -22,75 +27,80 @@ function Introduction({ isLoggedIn, toggleLoginModal }) {
           step.
         </p>
       </div>
-      <p>
-        Sign up below to help us understand your interest in getting a vaccine.
-        We’ll send you updates on how we’re providing vaccines across the
-        country—and when you can get your vaccine if you want one. We’ll also
-        offer information and answers to your questions along the way.
-      </p>
-
-      {isLoggedIn ? (
-        <Link className="usa-button" to="/form">
-          Sign up to stay informed
-        </Link>
-      ) : (
-        <>
-          <p>
-            When you sign in, we can fill in some of your information for you.
-          </p>
-          <p>
-            <button
-              type="button"
-              onClick={() => {
-                toggleLoginModal(true, 'coronavirus-vaccination');
-                recordEvent({
-                  event: 'cta-button-click',
-                  'button-type': 'default',
-                  'button-click-label': 'Sign in',
-                  'button-background-color': '#0071bb',
-                });
-              }}
-              className="usa-button"
-            >
-              Sign in
-            </button>
-            <Link
-              className="usa-button usa-button-secondary"
-              to="/form"
-              onClick={() => {
-                recordEvent({
-                  event: 'cta-button-click',
-                  'button-type': 'secondary',
-                  'button-click-label': 'Continue without signing in',
-                  'button-background-color': 'transparent',
-                });
-              }}
-            >
-              Continue without signing in
-            </Link>
-          </p>
-        </>
-      )}
+      <DowntimeNotification
+        appTitle="Covid 19 Vaccination Information"
+        dependencies={[externalServices.vetext_vaccine]}
```

Comment:
I think the dependency here should be in camel case -> `externalServices .vetextVaccine`

Reply:
Yep.  the devops config is with _ and I originally cut and pasted that.  thanks!